ibis/tests/test_version.py

```diff
@@ -8,13 +9,17 @@
 
 
 @pytest.mark.skipif(
-    bool(os.environ.get("CI")) or bool(os.environ.get("AZURECI")),
+    condition=bool(os.environ.get("CI") or os.environ.get("AZURECI")),
     reason="Testing import time on CI is flaky due to machine variance",
 )
 def test_import_time():
```

Comment:
I think this test is completely useless, but I'm going to revert the changes here as they are unnecessary.

Reply:
Reverted.